pkg/granted/credentialsprocess.go

```diff
@@ -0,0 +1,103 @@
+package granted
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/aws/smithy-go"
+	"github.com/common-fate/granted/pkg/cfaws"
+	"github.com/fatih/color"
+	"github.com/urfave/cli/v2"
+)
+
+/*
+* AWS Creds consumed by credential_process must adhere to this schema
+* https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-sourcing-external.html
+ */
+type AWSCredsStdOut struct {
+	Version         int    `json:"Version"`
+	AccessKeyID     string `json:"AccessKeyId"`
+	SecretAccessKey string `json:"SecretAccessKey"`
+	SessionToken    string `json:"SessionToken"`
+	Expiration      string `json:"Expiration"`
+}
+
+var CredentialsProcess = cli.Command{
+	Name:   "credentialsprocess",
+	Usage:  "",
+	Hidden: true,
+	Flags:  []cli.Flag{&cli.StringFlag{Name: "profile", Required: true}, &cli.StringFlag{Name: "url"}},
+	Action: func(c *cli.Context) error {
+
+		url := c.String("url")
+		profileName := c.String("profile")
+
+		if url == "" {
+			url = "http://localhost:3000/"
+		}
+		// In future we will support manual storing of url in config...
+		// conf, err := config.Load()
+		// if err != nil {
+		// 	log.Fatal("Failed to load Config for GrantedApprovalsUrl")
+		// }
+		// url = conf.GrantedApprovalsUrl
+		if url == "" {
+			log.Fatal("It looks like you haven't setup your GrantedApprovalsUrl\nTo do so please run: " + c.App.Name + " setup")
+		}
```

Comment:
This can be removed as per the comment above - let's gracefully fall back if the URL isn't provided so that this command still works.